<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>700</width><height>500</height></rect>
  </property>
  <property name="windowTitle"><string>Secure File Encryption</string></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="vLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">

      <!-- Encrypt Tab -->
      <widget class="QWidget" name="encryptTab">
       <layout class="QVBoxLayout" name="encryptLayout">
        <item>
         <widget class="QPushButton" name="selectFileEncrypt">
          <property name="text"><string>Select File to Encrypt</string></property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="questionEncrypt">
          <property name="text"><string>Decision Question...</string></property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="answerButtonsEncrypt">
          <item>
           <widget class="QPushButton" name="yesEncrypt">
            <property name="text"><string>Yes</string></property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="noEncrypt">
            <property name="text"><string>No</string></property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="encryptButton">
          <property name="text"><string>Encrypt</string></property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressEncrypt">
          <property name="minimum"><number>0</number></property>
          <property name="maximum"><number>0</number></property>
          <property name="visible"><bool>false</bool></property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="statusEncrypt">
          <property name="readOnly"><bool>true</bool></property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- Decrypt Tab -->
      <widget class="QWidget" name="decryptTab">
       <layout class="QVBoxLayout" name="decryptLayout">
        <item>
         <widget class="QPushButton" name="selectFileDecrypt">
          <property name="text"><string>Select File to Decrypt</string></property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="decryptButton">
          <property name="text"><string>Decrypt</string></property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressDecrypt">
          <property name="minimum"><number>0</number></property>
          <property name="maximum"><number>0</number></property>
          <property name="visible"><bool>false</bool></property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="statusDecrypt">
          <property name="readOnly"><bool>true</bool></property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- Compare Tab -->
      <widget class="QWidget" name="compareTab">
       <layout class="QVBoxLayout" name="compareLayout">
        <item>
         <layout class="QHBoxLayout" name="compareControls">
          <item>
           <widget class="QPushButton" name="selectFileCompare">
            <property name="text"><string>Select File for Comparison</string></property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runCompare">
            <property name="text"><string>Run Comparison</string></property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="fileInfoLayout">
          <item row="0" column="0">
           <widget class="QLabel">
            <property name="text"><string>Path:</string></property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="filePathCompare"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel">
            <property name="text"><string>Size:</string></property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="fileSizeCompare"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="compareTable">
          <property name="rowCount"><number>2</number></property>
          <property name="columnCount"><number>3</number></property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="loadingCompare">
          <property name="minimum"><number>0</number></property>
          <property name="maximum"><number>0</number></property>
          <property name="visible"><bool>false</bool></property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusCompare"/>
        </item>
       </layout>
      </widget>

     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
